### DISCUSSION TITLE
How to add author info to docs?

### DISCUSSION THREAD

**Discussion by zephyr-sh** [facebook/docusaurus#10701] [Q&A] [answered]
![quest](https://github.com/user-attachments/assets/6e280528-fa6e-49a4-b3e9-89296874deaf)

Hello everyone, as shown in the image above. I’ve tried searching for related discussions, but I couldn’t find anything relevant in a short time. I’d like to ask how to implement this feature? Are there any existing solutions available? Or do I need to implement it myself?

**Comment by slorber** [comment, marked as answer]
The docs plugin doesn't have support for "authors" like the blog has.

But you have the freedom to implement this yourself.
You can swizzle the docs component, use custom front matter of your choice, and render that front matter in the swizzled docs components.

**Reply by zephyr-sh** [reply]
Got it, thank you! This way I can make sure I won't reinvent the wheel.

**Comment by zephyr-sh** [comment]
The following is my simple implementation:

https://docsaid.org/en/blog/customized-docusaurus-author-to-plugin-content-docs

I hope it can help those in need.

**Reply by leshicodes** [reply]
I’m gonna try this out on my pilot site. This could be awesome for me if it works.
